TestModels/RecursiveShape/src/Index.dfy

```diff
@@ -0,0 +1,30 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+include "SimpleRecursiveShapeImpl.dfy"
+
+module {:extern "simple.recursiveshape.internaldafny"} SimpleRecursiveShape refines AbstractSimpleRecursiveShapeService {
```

Comment:
Let's not add `extern` anymore since it is to be deprecated.

Reply:
Done